### PR TITLE
Updating missing info in feature guides.

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Docs: Updating missing info in feature guides. Closes cksource/ckeditor5-internal#2655

---

### Additional information

_Merge together with cksource/ckeditor5-internal#2666 and https://github.com/cksource/collaboration-features/pull/4987_

### CODE REVIEW

File: packages/ckeditor5-upload/docs/features/simple-upload-adapter.md [hunk 1]
```diff
@@ -13,8 +13,7 @@ See the [Server–side configuration](#server-side-configuration) section to lea
 ## Installation
 
 <info-box info>
-	The {@link module:upload/adapters/simpleuploadadapter~SimpleUploadAdapter simple upload adapter} plugin is not available out–of–the–box in any of {@link installation/getting-started/predefined-builds#available-builds official editor builds}. Check out the {@link installation/plugins/installing-plugins "Installing plugins" guide} to learn more.
-</info-box>
+	This feature is not available in any of {@link installation/getting-started/predefined-builds#available-builds official editor builds}.  See the [installation](#installation) section to learn how to enable it in your editor.
```
Comment (apadol-cksource) [inline packages/ckeditor5-upload/docs/features/simple-upload-adapter.md:16]: Let's use "predefined builds" everywhere.

**Review by apadol-cksource — CHANGES_REQUESTED** [review]

File: packages/ckeditor5-markdown-gfm/docs/features/markdown.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ The Markdown plugin allows switching the default CKEditor 5 output from HTML to
 Please remember that Markdown syntax is very simple and it does not cover all the rich-text features. Some features provided by CKEditor 5 will thus work as intended only when output to HTML as they have no Markdown equivalent.
 
 <info-box info>
-	You can learn more about the possible practical applications of Markdown editing with CKEditor 5 in [this dedicated blog post depicting the idea, solutions and a case study](https://ckeditor.com/blog/CKEditor-5-the-best-open-source-Markdown-editor/).
+	This feature is not available in any of {@link installation/getting-started/predefined-builds#available-builds official editor builds}.  See the [installation](#installation) section to learn how to enable it in your editor.
```
[the remark this answers is not in the document]
Comment (godai78) [reply packages/ckeditor5-markdown-gfm/docs/features/markdown.md:11]: Not really, this info was duplicated from lead-in as an infobox, and hence was removed. It is not available again in the demo section.

File: packages/ckeditor5-style/docs/features/style.md [hunk 3]
```diff
@@ -10,12 +10,20 @@ The style feature lets the user apply pre-configured styles to existing elements
 
 Under the hood, every style applies one or more HTML classes to such an element which, depending on your integration requirements, can be used to either control the visual styles of that element or apply additional semantics. <!-- The {@link features/remove-format remove formatting} feature can be used to clear those styles. -->
 
+<info-box info>
+	This feature is not available in any of {@link installation/getting-started/predefined-builds#available-builds official editor builds}.  See the [installation](#installation) section to learn how to enable it in your editor.
```
[the remark this answers is not in the document]
Comment (godai78) [reply packages/ckeditor5-style/docs/features/style.md:14]: Fixed in commit.

**Review by apadol-cksource — APPROVED** [review]
LGTM

**Pull request merged**